### DISCUSSION TITLE
ITAR NASA Applications for CFS

### DISCUSSION THREAD

**Discussion by JudeLabs** [nasa/cFS#480] [General]
Are there other Open Source applications that NASA maintains  for CFS that may be ITAR, but are available for appropriate projects where NASA is a customer?

**Comment by skliper** [comment]
NASA is restricted from open sourcing anything that is ITAR.  There is NASA internal apps and related software that can be shared via other mechanisms though as long as the appropriate agreements are in place (software usage agreement, interagency agreement, etc).  Feel free to reach out through your NASA contacts or contact cfs-program@lists.nasa.gov directly for more details.

**Reply by JudeLabs** [reply]
Ok great thanks,  yea "Open Source" was a misnomer.  I meant apps assigned an ITAR designation that are available to appropriate projects that are allowed to work on such applications.

**Reply by skliper** [reply]
Heads up I just submitted a PR to update the cFS README for all the open source cFS related repos in the nasa org I could find. https://github.com/nasa/cFS/pull/481
